src/app/staticanalysis/modules/functionCallGraph.js

```diff
@@ -16,6 +16,30 @@ function buildLocalFuncCallGraphInternal (functions, nodeFilter, extractNodeIden
   return callGraph
 }
 
+/**
+ * Builds a function call graph for the current contracts.
+ * Example Contract call graph:
+ *
+ * {
+ *  "KingOfTheEtherThrone": {
+ *    "contracts": {...},                                        // Contract node as defined in abstractAstView.js
+ *    "functions": {
+ *      "KingOfTheEtherThrone.claimThrone(string memory)": {    // function in KingOfEtherThrone
```

Comment:
Thanks for the reply. Ok, I see. Concerning the source code order. At the moment i only need the order of contracts because if some contract inherits from another i copy the state variables from the base contract. I can quick fix it by postpone the logic to run after i built all contract objects. The whole thing with import aliases is probably a bit more complicated and i don't have all the info in the AST at the moment, this is the AST node returned by the compiler for the import `{"attributes":{"file":"file1"},"id":3,"name":"ImportDirective","src":"0:20:1"}` the alias is missing. That means i can not resolve stuff like `contract c is x.c {}` because the alias is not available in the AST. For now i could produce warnings if i spot two contracts with the same name, that analysis does not support that at the moment(should be a rare case anyways...), if the AST contains all the info we can think of implementing this properly but i think it would take some time. What do you think?